src/v2/Apps/Conversation/Components/Details.tsx

```diff
@@ -40,11 +40,11 @@ const DetailsContainer = styled(Flex)<{ opacity?: 0 | 1; transform?: string }>`
   ${DETAIL_BOX_ANIMATION}
   ${media.xl`
     transform: ${({ transform }: { transform?: string }) => transform};
-    ${DETAIL_BOX_MD_ANIMATION}
+    ${DETAIL_BOX_LG_ANIMATION}
     z-index: 0;
   `}
-  ${media.xs`
-    ${DETAIL_BOX_XS_ANIMATION}
+  ${media.md`
+    ${DETAIL_BOX_MD_ANIMATION}
```

Comment:
Adjusted this breakpoint to include medium sized screens (certain sized tablets)

Before:
![Screen Recording 2020-07-22 at 3 49 34 PM](https://user-images.githubusercontent.com/12748344/88230536-9354c180-cc40-11ea-95ed-1783831e1dba.gif)

<br>
<br>




After: